<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Disconnector</class>
 <widget class="QWidget" name="Disconnector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="textDiscOff">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отключено</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rs.qrc">:/Resources/Icon/CircuitBreakerOn.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nameCell">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameDisconnector">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="ClickableLabel" name="offCircuit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textDiscOm">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Включено</string>
   </property>
  </widget>
  <widget class="ClickableLabel" name="onCircuit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rs.qrc"/>
 </resources>
 <connections/>
</ui>
